packages/compass-e2e-tests/smoke-test.ts

```diff
@@ -122,9 +161,7 @@ async function readPackageVersion(packagePath: string) {
 }
 
 async function run() {
-  const parsedArgs = argv.parseSync();
-
-  const context = parsedArgs as SmokeTestsContext;
+  const context: SmokeTestsContext = argv.parseSync();
```

Comment:
Ideally I would have either used `satisfies` here (but our prettier's typescript parser needs an update to support that) or simply deleted the type now that it matches the expected type.

```suggestion
  const context = argv.parseSync() satisfies SmokeTestsContext;
```